modules/javascript/pages/ionic.adoc

```diff
@@ -1,77 +1,34 @@
 = Ionic
 :page-partials:
-:page-edition: 
+:page-edition: Public Preview
 :page-toclevels: 2@
-:description: Using Couchbase Lite with Javascript Ionic applications
+:description: Couchbase is excited to announce the public preview of the Ionic Capacitor plugin for Couchbase Lite, which allows developers to leverage Couchbase Lite within cross-platform applications in JavaScript for iOS and Android. This plugin is open-source and supported by the Couchbase community. 
 
 include::partial$_set_page_context_for_javascript.adoc[]
 
-:param-abstract: If you are building mobile or desktop applications using web technologies such as JavaScript/HTML/CSS, there are options for using Couchbase Lite as an embedded database within your app.
-:param-related: xref:cordova.adoc[] | xref:react.adoc[]
-include::{root-partials}_show_page_header_block.adoc[]
+== Project Overview
 
-.Support Changes
-[IMPORTANT]
+NOTE: Developers who are currently using the https://ionic.io/docs/couchbase-lite[previous version of the Ionic plugin] should migrate to the newer version of the plugin documented https://cbl-ionic.dev[here].
 
---
-* The Ionic plugin is now open source and community supported.
-* The Couchbase Lite API, which contains storage and sync functionality, is fully supported as normal for Enterprise customers. 
---
+This public preview release enables developers using Ionic to integrate Couchbase Lite into their mobile applications, ensuring they can build offline-first, cross-platform solutions with ease.
 
-== Introduction
+== Important Notes
 
+* Platform Support - The plugin currently supports iOS and Android platforms.
 
-// tag::summary[]
-Ionic's Couchbase Lite integration is an open source solution for web developers, enabling easy to build secure, high-performance, offline-enabled apps.
+* Community-Supported - This open-source plugin is Couchbase community supported and maintained, and it requires an Enterprise Edition Couchbase Lite license.
+While it is not officially supported by Couchbase, any issues specific to Couchbase Lite are covered under our existing support policy.
 
-// end::summary[]
-This integration supports apps built for iOS, Android, and native Windows.
+NOTE: Couchbase Community Supported is an Open source integration that is developed by Couchbase. Issues can also be raised in the relevant GitHub repositories or via Couchbase forums or Discord. Support is provided best effort by development team.
```

Comment:
Is `Couchbase Community Supported` an `integration` or is it a support scheme?

Reply:
I believe it's a support scheme, `Couchbase Community Supported` is the term for an open source integration with best effort dev support on our end.